packages/core/src/App/Containers/NotificationsDialog/notification-list-wrapper.tsx

```diff
@@ -0,0 +1,61 @@
+import classNames from 'classnames';
+import React, { LegacyRef } from 'react';
+import { Text, ThemedScrollbars } from '@deriv/components';
+import { routes } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import { Localize } from '@deriv/translations';
+import EmptyNotification from 'App/Components/Elements/Notifications/empty-notification';
+import NotificationsClearAllFooter from './notifications-clear-all-footer';
+import NotificationsList from './notifications-list';
+
+type TNotificationListWrapper = { clearNotifications: () => void };
+
+const NotificationListWrapperForwardRef = React.forwardRef(
+    ({ clearNotifications }: TNotificationListWrapper, ref: LegacyRef<HTMLDivElement> | undefined) => {
+        const { notifications, ui } = useStore();
+        const { is_notifications_empty } = notifications;
+        const { is_mobile } = ui;
+
+        const traders_hub = window.location.pathname === routes.traders_hub;
+
+        return (
+            <div
+                data-testid='dt_notifications_list_wrapper'
+                className={classNames('notifications-dialog', {
+                    'notifications-dialog--pre-appstore':
+                        traders_hub || window.location.pathname.startsWith(routes.account),
+                })}
+                ref={ref}
+            >
+                <div className='notifications-dialog__header'>
+                    <Text
+                        as='h2'
+                        className='notifications-dialog__header-text'
+                        weight='bold'
+                        color='prominent'
+                        styles={{
```

Comment:
Text component supports line_height. U can replace inline style with that

Reply:
the defined line height values doesnt have units, and while setting it using props, the footer is getting misaligned.
maybe if required, we can look into it in a separate card.
![Screenshot 2023-09-01 at 7 02 46 PM](https://github.com/binary-com/deriv-app/assets/122449658/9e90c42f-a84f-45f5-8e0d-75b19db3bbb7)